README.md

```diff
@@ -1,22 +1,42 @@
-# dangermattic
-Shared collection of Danger plugins
+# Dangermattic
+`Dangermattic` builds on [Danger on Ruby](https://danger.systems/ruby/) and is essentially a collection of Danger plugins. Its goal is to provide customisable checks and common utilities to help perform checks on Pull Requests, from simple routine validations to more sophisticated ones.
 
 ## Installation
 
-Add to your `Gemfile`
+Add to your project's `Gemfile`
 ```
-gem 'dangermattic'
+gem 'danger-dangermattic', git: 'https://github.com/Automattic/dangermattic'
 ```
 
-## Usage
+## Available plugins and their usage
 
-    Methods and attributes from this plugin are available in
-    your `Dangerfile` under the `dangermattic` namespace.
+Once the main Gem is installed, all Dangermattic plugins are available in your `Dangerfile` under their corresponding namespace:
+
+- `manifest_pr_checker` - Plugin to check if changes on a manifest file (i.e. `Gemfile`, `Podfile`) has a corresponding change in a lock file (i.e. `Gemfile.lock`, `Podfile.lock`)
+```ruby
+# Reports a warning if the Gemfile was changed but the Gemfile.lock wasn't
+manifest_pr_checker.check_gemfile_lock_updated
+```
+- `milestone_checker` - Plugin for performing checks on a milestone associated with a pull request
+```ruby
+# Reports a warning if the Gemfile was changed but the Gemfile.lock wasn't
+milestone_checker.check_milestone_due_date(days_before_due: 3)
+```
+- `pr_size_checker` - Plugin to check the size of a Pull Request content and text body
+```ruby
+# Reports a warning if a pull request diff size is greater than 300
+pr_size_checker.check_diff_size(max_size: 300)
+```
+- `view_changes_need_screenshots` - Detects view changes in a PR and reports a warning if there are no attached screenshots
+```ruby
+# Reports a warning if a pull request changing views doesn't have a screenshot
+view_changes_need_screenshots.view_changes_need_screenshots
+```
 
 ## Development
 
-1. Clone this repo
-2. Run `bundle install` to setup dependencies.
-3. Run `bundle exec rake spec` to run the tests.
-4. Use `bundle exec guard` to automatically have tests run as you make changes.
-5. Make your changes.
+- Clone the repo and run `bundle install` to setup dependencies
+- Run `bundle exec rake spec` to run the all the tests, RuboCop and Danger Lint
```

Comment:
Seems strange to me that the Rake task is named `spec` but in practice it does more than just running the ruby specs (aka unit tests) but also runs linters.

I'd suggest to update your `Rakefile` (and then this README portion accordingly) to have:
 - `bundle exec rake` to run all the tests, rubocop and Danger lint
 - `bundle exec rake spec` to run all the tests (aka specs). Which in practice should be the same as `bundle exec rspec`. I think this one might be automatically added to your `Rakefile` when you `require 'rspec/core/rake_task'` actually?
 - `bundle exec rake lint` to run both rubocop and danger lint.

Reply:
Ah indeed, that's something I got from Danger examples / templates. Thanks for noticing and the suggestions!
I've reviewed the `Rakefile` and updated the `README.md`: 87fe26d65a9e8b121e5134f8fde60db7a13113d6